### ISSUE TITLE
[Sass] UNCSS Stripping Dynamicly Added Styles

### ISSUE THREAD

**Issue by EHackettTriCore** [zurb/foundation-sites#7598]
**How can we reproduce this bug?**

1. Add CSS selectors of any classes dynamically added through Javascript to your scss files.

**What did you expect to happen?**
All CSS/SCSS I added outside of the Foundation framework would make it to distribution.

**What happened instead?**
Without knowing that UNCSS was being used during the distribution process, dynamically added classes worked in `npm start` but not in `npm run build`. Only after tracking down and learning about uncss did I realize what was happening.

**Test case**
Any dynamically added classes in javascript that were defined in the SCSS files will work in `npm start` but will be stripped during the building process.

**Solution Ideas**

A. Add documentation to the `SASS` page instructing users to use the `.is-` CSS naming conventions when working with dynamically added classes to maintain naming consistency with Foundation and so the styles aren't stripped during distribution process. And add a note that they can add their own custom class prefixes to gulp file like `.js-` or something if they want to be more advanced.

B. By default scope UNCSS to only look at Foundation CSS. Generally I think we could assume anything a user adds on top of Foundation is likely going to be used in production. If they're an advanced user perhaps there could be an option for Global/Core to adjust UNCSS scoping.

C. Create a `Build Process` page in the documentation that outlines the building process more theroughly.

---

I like UNCSS however, it caused me a bit of a headache and wasted time tracking down what was happening to my css between development and distribution. I can see this being a problem for many  beginner/moderate Foundation developers.

**Comment by gakimball** [comment]
Duplicate of #7473
